scripts/install.sh

```diff
@@ -91,17 +99,17 @@ Environment="PATH=$PATH"
 [Install]
 WantedBy=default.target
 EOF
-    SYSTEMCTL_RUNNING="$(systemctl is-system-running || true)"
-    case $SYSTEMCTL_RUNNING in
-        running|degraded)
-            status "Enabling and starting ollama service..."
-            $SUDO systemctl daemon-reload
-            $SUDO systemctl enable ollama
-
-            start_service() { $SUDO systemctl restart ollama; }
-            trap start_service EXIT
-            ;;
-    esac
+        $SUDO systemctl daemon-reload
+        $SUDO systemctl enable ollama
```

Comment:
This will fail if systemd isn't running. It also doesn't start the service which I'm not sure is intentional